cmd/image-builder/main.go

```diff
@@ -60,7 +60,7 @@ func main() {
 		logrus.Warn("Sentry/Glitchtip was not initialized")
 	} else {
 		sentryhook := sentrylogrus.NewFromClient([]logrus.Level{logrus.PanicLevel,
-			logrus.FatalLevel, logrus.ErrorLevel},
+			logrus.FatalLevel, logrus.ErrorLevel, logrus.InfoLevel},
```

Comment:
This seems very overkill, the point of glitchtip is to only get the few errors that are so rare they might not influence the SLOs.

Reply:
the `InfoLevel` part will be reverted later, that's why it's a separate commit, only meant for testing to help us make sure everything is working.